ONSdigital/eq-survey-runner
# Clean up

**_What**_

Minor code refactors to make code (hopefully) easier to understand and to reinforce the structure.  Mostly concerned with how variables in the main blueprint are named and then refactoring the pattern library into a blueprint which extends the app in a self contained way

**_How to test**_

1) Check out the app
2) Run unit tests
3) Verify nothing is broken

**_Who can test**_

Anyone except @weapdiv-david
